### PR TITLE
Add works and about section

### PR DESCRIPTION
**Pull request by kwekubright**
### Description
In the feature i have done the following:

- Added works section to index.html
- Added about section to index.html
- Added flexbox and grid style for works and about section
- Added screenshot of works and icons of JavaScript, HTML and CSS

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -58,6 +58,161 @@ <h3>Let's connect</h3>
 
     </section>
 
+    <section class="works">
+      <div class="grid works-grid-container wrap padding-top-0">
```
Comment (prolajumokeoni) [inline index.html:62]: - [ ] Kindly add all button interactions for the see project  buttons in this section 👍🏿 

Screenshot_34

File: index.html [hunk 2]
```diff
@@ -58,6 +58,161 @@ <h3>Let's connect</h3>
...
+              <img class="social github" src="images/icons/github-icon.png" alt="Twitter icon">
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <img class="social f" src="images/icons/f-icon.png" alt="Twitter icon">
+            </a>
+          </li>
+        </ul>
+        <button type="button" class="button button-enabled">See my resume</button>
```
Comment (prolajumokeoni) [inline index.html:175]: - [ ] Kindly add button interactions for the see my resume button in this section 👍🏿 

Screenshot_34

File: index.html [hunk 3]
```diff
@@ -58,6 +58,161 @@ <h3>Let's connect</h3>
...
+
+        <div class="other-info">
+          <ul class="flex other-info-wrap padding-0">
+            <li class="other-info-list">
+              <div class="flex li-flex wrap">
+                <h4>Language</h4>
+                <span> > </span>
```
Comment (prolajumokeoni) [inline index.html]: - [ ] Kindly add the right icon to this header tag,so your design look exactly like the figma design  👍🏿 



Figma Design          |  Your design    
:-------------------------:|:-------------------------:
Screenshot_35 | Screenshot_33

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @kwekubright, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used


## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Danieldotcomcoder — APPROVED** [review]
Hello @kwekubright, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 👍 👍 😄 👏 

Congratulations! 🎉 🎉 🎉 

comp-2
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**